tools/create_service_bundle.py

```diff
@@ -0,0 +1,568 @@
+#!/usr/bin/env python3
+#
+# TODO: usage, description.
+
+from datetime import date, datetime
+from typing import List
+import json
+import logging
+import os
+import re
+import retrying
+import sys
+
+from sdk_utils import groupby
+import sdk_cmd
+import sdk_diag
+import sdk_utils
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO, format="%(message)s")
```

Comment:
Please include a timestamp here, it can be very valuable at times.

Reply:
Timestamp is available with the latest changes.